internal/pkg/cli/svc_deploy_test.go

```diff
@@ -174,6 +177,16 @@ func TestSvcDeployOpts_Execute(t *testing.T) {
 
 			wantedError: fmt.Errorf("get available features of the prod-iad environment stack: some error"),
 		},
+		"error out if force deploy for static site service": {
+			inForceFlag: true,
+			inSvcType:   manifestinfo.StaticSiteType,
```

Comment:
isn't `svcType` figured out by getting the service info from SSM? Do you think we can add a `store` mock here and mock return `GetService`?

Reply:
i feel `GetService` is not the main thing we want to test. we could assume the service is properly configured in SSM (whose format is probably what we won't change). And if that is the case, most likely for every test case we'll have `GetService` which I feel cumbersome and distracting